src/live_restore/live_restore_fs.c

```diff
@@ -625,6 +627,8 @@ __live_restore_fh_write_int(
     WT_ASSERT_ALWAYS(session, __wt_rwlock_islocked(session, &lr_fh->lock),
       "Live restore lock not taken when needed");
     WT_RET(__live_restore_fh_write_destination(session, lr_fh, offset, len, buf));
+    if (background_thread)
```

Comment:
Can we increment this in `__wti_live_restore_fs_restore_file` to avoid the bool argument?

Reply:
We don't know if the write is successful at that point, it is basically guaranteed to be though but I felt like it made more sense doing it after the write had happened. What do you think?

Reply:
You could increment it after the call to __live_restore_fill_hole, it's guaranteed by that point